pkg/kubelet/preemption/preemption_test.go

```diff
@@ -231,9 +235,17 @@ func TestGetPodsToPreempt(t *testing.T) {
 			expectErr:             false,
 			expectedOutput:        []*v1.Pod{allPods[highRequestBurstable], allPods[highRequestGuaranteed]},
 		},
+		{
+			testName:              "evict cluster critical pod for node critical pod",
```

Comment:
can we also add a case where we don't evict a node critical for a cluster critical pod?

Reply:
Done :)